metadata/lang/es.yml

```diff
@@ -76,5 +77,5 @@ presenter-notes-explanation: |
 #This window is linked to the main window. Changing slides on one will cause the slide to change on the other.
 #
 #Useful when presenting.
-plaintext-slides: "O visualiza las diapositivas en modo solo texto sin JS" # Or view the plaintext slides without JS
+plaintext-slides: "O visualiza las diapositivas en modo solo texto sin JS" # Plain-text slides
```

Comment:
@beatrizserrano: in english we streamlined this text to simply "Plain-text slides" ..do you want to do the same for the Spanish, or leave it as it is?